### PR TITLE
Adds passage for update assignment

### PR DESCRIPTION
**Pull request by vly-smartsheet**
- An end-user was having trouble utilizing assignment updates from the API.
- The API docs didn't contain a passage for Assignment Updates
- This PR introduces an example to update an assignment through the API
- Nick Cox had mentioned in a slack thread that assignments in the UI are updated as a delete then create.
- I've tested out concerns pertaining to the activity feed and haven't seen anything unusual.

References
----------

- [Slack thread](https://smartsheet.slack.com/archives/C03M9TW8C56/p1667241968584049)

Description
-----------

Testing
-------

1. Pull branch
2. Run `bundle exec middleman server`
3. Verify the integrity of the information and call out if known issue should be added.

Risk
----

- [x] Low
- [ ] Medium
- [ ] High

They're docs, but if something needs to be mentioned it should be added.

Deployment Checklist
--------------------

_Provide any details that would help ensure the deployment readiness of this change._

- [x] Testing details are reviewed and executed (preferably by multiple people)
- [x] Risks described are reviewed and mitigated
- [x] Rollback is available
- [ ] Service health will be monitored after deployment is complete

_Provide details if any of the following is checked._

- [ ] Requires manual deployment steps
- [ ] Requires non-standard deployment steps
- [ ] Requires downtime

Dependencies
------------

- Provide links to PRs or other dependencies.

Checklist
---------

Authors should use the checklist below to understand readiness of the pull request for review

- [x] Follows the [contributing guidelines](https://github.com/APX/benchpress-server/blob/master/CONTRIBUTING.md)
- [ ] Has been added to its JIRA card
- [x] Has been assigned to one or more reviewers to review
- [x] Includes a description
- [x] Adheres to design specifications
- [x] Includes testing instructions
- [x] Includes spec coverage (where applicable)
- [x] Includes automation pass rate and link
- [ ] Includes a description of database changes (where applicable)
- [x] Includes dependencies (if applicable)
- [x] Deployment checklist has been reviewed and updated
- [x] New library added and license compliance has been review

### CODE REVIEW

File: source/includes/_assignments.md [hunk 1]
```diff
@@ -88,6 +88,15 @@ POST /api/v1/users/<user_id>/assignments
                  'https://api.rm.smartsheet.com/api/v1/users/<user_id>/assignments?auth=...'
 ```
 
+## Update an Assignment
+
+Typical parameters: `starts_at`, `ends_at`, `allocation_mode (percent, hours_per_day, fixed)`, `percent`, `hours_per_day`, `fixed_hours`, `status_option_id`, `description`, `note`
+
+```sh
+curl -XPUT -d 'description=<description>&percent=<DECIMAL>&starts_at=<YEAR-MO-DAY>' \
+                'https//api.rm.smartsheet.com/api/v1/assignments/<assignment_id>/assignments?auth=...'
```
Comment (tithonium) [inline source/includes/_assignments.md]: Got an extra /assignments at the end there

**Review by tithonium — CHANGES_REQUESTED** [review]

File: source/includes/_assignments.md (hunk 1 above)
Comment (vly-smartsheet) [reply source/includes/_assignments.md]: oops, that's embarrassing 😶‍🌫️

**Comment by vly-smartsheet** [comment]
The change request persists after the change has been made 😭

**Review by baonguyen84 — APPROVED** [review]

File: source/includes/_assignments.md [hunk 2]
```diff
@@ -88,6 +88,15 @@ POST /api/v1/users/<user_id>/assignments
...
 
+## Update an Assignment
+
+Typical parameters: `starts_at`, `ends_at`, `allocation_mode (percent, hours_per_day, fixed)`, `percent`, `hours_per_day`, `fixed_hours`, `status_option_id`, `description`, `note`
+
+```sh
+curl -XPUT -d 'description=<description>&percent=<DECIMAL>&starts_at=<YEAR-MO-DAY>' \
+                'https//api.rm.smartsheet.com/api/v1/assignments/<assignment_id>?auth=...'
+```
+
```
Comment (aiyer-smartsheet) [inline source/includes/_assignments.md:99]: @vly-smartsheet There was a question in the slack thread: if `update` is tracked by ENS (activity feed), do we know the answer to that?

Comment (vly-smartsheet) [reply source/includes/_assignments.md:99]: If it's the one that's in the referenced in the references for this PR from #ask-RM-Eng it looks like it's being tracked and there's nothing off when I look at it.

/Me Page:
Screen Shot 2022-11-02 at 9 23 13 AM

/Projects/Activity page:
Screen Shot 2022-11-02 at 9 25 06 AM

Email client:
Screen Shot 2022-11-02 at 9 25 56 AM

**Review by aiyer-smartsheet — APPROVED** [review]

**Review by tithonium — APPROVED** [review]

**Pull request merged**